daemon/internal/builder-next/executor_linux.go

```diff
@@ -6,6 +6,7 @@ import (
 	"path/filepath"
 	"strconv"
 
+	containerdoci "github.com/containerd/containerd/v2/pkg/oci"
```

Comment:
```suggestion
	coci "github.com/containerd/containerd/v2/pkg/oci"
```

We tend to use the `coci` alias for this one